src/commands/music/index.js

```diff
@@ -0,0 +1,215 @@
+import Promise from 'bluebird';
+import R from 'ramda';
+import ytdl from 'ytdl-core';
+
+import { addSong, delSong, delSongs, getSong, getSongs, getNextSong } from '../../redis';
+import { time } from '../../helpers';
+import T from '../../translate';
+
+let timer;
+
+
+function vleave(client, evt, lang) {
+  const info = client.VoiceConnections.getForGuild(evt.message.guild);
+  if (info) {
+    info.voiceConnection.disconnect();
+    delSongs(evt.message.guild.id);
+    delSong(evt.message.guild.id);
+  } else {
+    return Promise.resolve(T('music_not_connected', lang));
+  }
+}
+
+function vjoin(client, evt, suffix, lang) {
+  if (!suffix) {
+    const channel = evt.message.guild.voiceChannels.filter(vc => vc.members.map(m => m.id).indexOf(evt.message.author.id) > -1);
+    if (channel.length > 0) {
+      channel[0].join();
+      timer = setTimeout(vleave, 1200000, client, evt);
+    } else {
+      return Promise.resolve(T('music_not_found', lang));
+    }
+  } else {
+    const channel = evt.message.guild.voiceChannels.find(c => c.name.toLowerCase() === suffix.toLowerCase());
+    if (channel) {
+      channel.join();
+      timer = setTimeout(vleave, 1200000, client, evt);
+    } else {
+      return Promise.resolve(T('music_not_found', lang));
+    }
+  }
+}
+
+function play(client, evt, music, lang) {
+  const info = client.VoiceConnections.getForGuild(evt.message.guild);
+  if (!info) return Promise.resolve(T('music_not_connected', lang));
+  const encoder = info.voiceConnection.createExternalEncoder({
+    type: 'ffmpeg',
+    realtime: true,
+    source: music.url,
+    outputArgs: ['-compression_level', 7]
```

Comment:
Ah, yeah, it doesn't seem to have compression level, it's probably only for videos.
[https://github.com/qeled/discordie/blob/master/lib/voice/players/FFmpegEncoder.js#L186](https://github.com/qeled/discordie/blob/master/lib/voice/players/FFmpegEncoder.js#L186)